app/src/main/java/org/jellyfin/androidtv/base/ClickablePresenter.kt

```diff
@@ -0,0 +1,7 @@
+package org.jellyfin.androidtv.base
+
+import androidx.leanback.widget.Presenter
+
+abstract class ClickablePresenter : Presenter() {
+	abstract fun onItemClicked(item: Any?)
```

Comment:
Might be worth using a generic on the class/interface so we don't need to use the `any` type

Reply:
Doesn't usage of generics yield completely different classes? In C# at least it does iirc. If so that mean as a consequence, that we can no longer call the `onItemClicked` method on a common parent type, no?

Reply:
I don't know how C# handles and have no idea if it actually is possible. If it does it will make the code more typesafe as I don't like having to cast "Any" to the objects we expect. Changes are it will introduce issues later.

Reply:
While I agree that it would be nice, I currently don't see a way to make it happen. I think somewhere a down cast has to happen, as we are given an Any for item. Maybe I'm too sleepy to see it right now though. Can you please have a look tomorrow as well?